packages/utils/src/mintReceiptNFT.ts

```diff
@@ -271,6 +271,8 @@ export async function mintAndSaveReceiptNFT({
     // #region: Mint NFT on current chain only
     const receiptContract = currentChain.contracts.receiptMintbotERC721
     console.log("CTR", receiptContract)
+
+    // WARN: XRPL doesn't use contracts
```

Comment:
I assume we need it to do something different if it's XRPL? Maybe have it invoke a separate function?